src/code/z_bgcheck.c

```diff
@@ -4313,7 +4307,7 @@ s32 WaterBox_GetSurface2(PlayState* play, CollisionContext* colCtx, Vec3f* pos,
  * WaterBox get BgCam index
  */
 u32 WaterBox_GetBgCamIndex(CollisionContext* colCtx, WaterBox* waterBox) {
-    u32 bgCamIndex = WATERBOX_BGCAM_INDEX(waterBox->properties);
+    u32 bgCamIndex = waterBox->properties & 0xFF;
```

Comment:
what happened to the macro here...

Reply:
I added them in https://github.com/zeldaret/oot/pull/1288 ,
but then doing the same in https://github.com/zeldaret/oot/pull/1293 with `SurfaceType.data` met some backlash so I reverted them in that same PR https://github.com/zeldaret/oot/pull/1293/commits/3ad3605cc9fe712df39a944ea037aa7584c2ce4d
So I guess we don't want them idk